bin/get_credentials

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
```

Comment:
Usage 

```
ruby get_credentials newrelic
license=44e99c4dec77edb10c604eb7db719610ae4c26eb
```

`ruby get_credentials newrelic ` this will output all the services that has either the tag ```newrelic ``` or name as ```newrelic```